worker/migrations/migrate-accounts-to-organisation.go

```diff
@@ -0,0 +1,282 @@
+	if username == "" && email != "" {
+		username = email
+	}
+
+	key := orgKey[:32]
+	hmac := orgKey[32:]
+
+	ivURL := crypto.GenerateRandomBytes(16)
+	encURL, err := crypto.EncryptWithAES256HMAC(key, hmac, []byte(url), ivURL)
+	if err != nil {
+		return nil, err
+	}
+	u := bitwarden.LoginURI{URI: encURL, Match: nil}
+	uris := []bitwarden.LoginURI{u}
+
+	ivName := crypto.GenerateRandomBytes(16)
+	encName, err := crypto.EncryptWithAES256HMAC(key, hmac, []byte(manifest.Name), ivName)
+	if err != nil {
+		return nil, err
+	}
+
+	ivUsername := crypto.GenerateRandomBytes(16)
+	encUsername, err := crypto.EncryptWithAES256HMAC(key, hmac, []byte(username), ivUsername)
+	if err != nil {
+		return nil, err
+	}
+
+	ivPassword := crypto.GenerateRandomBytes(16)
+	encPassword, err := crypto.EncryptWithAES256HMAC(key, hmac, []byte(password), ivPassword)
+	if err != nil {
+		return nil, err
+	}
+
+	login := &bitwarden.LoginData{
+		Username: encUsername,
+		Password: encPassword,
+		URIs:     uris,
+	}
+
+	md := metadata.New()
+	md.DocTypeVersion = bitwarden.DocTypeVersion
+
+	bitwardenFields := make([]bitwarden.Field, 0)
+
+	for name, rawValue := range auth {
+		value, ok := rawValue.(string)
+		if !ok {
+			continue
+		}
+		if !isAdditionalField(name) {
+			continue
+		}
+
+		ivName := crypto.GenerateRandomBytes(16)
+		encName, err := crypto.EncryptWithAES256HMAC(key, hmac, []byte(name), ivName)
+		if err != nil {
+			return nil, err
+		}
+
+		ivValue := crypto.GenerateRandomBytes(16)
+		encValue, err := crypto.EncryptWithAES256HMAC(key, hmac, []byte(value), ivValue)
+		if err != nil {
+			return nil, err
+		}
+
+		logger.Infof("Adding field %s = %s", name, value)
+		field := bitwarden.Field{
+			Name:  encName,
+			Value: encValue,
+			Type:  bitwarden.FieldTypeText,
+		}
+		bitwardenFields = append(bitwardenFields, field)
+	}
+
+	c := bitwarden.Cipher{
+		Type:           bitwarden.LoginType,
+		Name:           encName,
+		Login:          login,
+		SharedWithCozy: true,
+		Metadata:       md,
+		Fields:         bitwardenFields,
+	}
+	return &c, nil
+}
+
+func getCipherLinkFromManifest(manifest *app.KonnManifest) (string, error) {
+	var link string
+	if manifest.VendorLink == nil {
+		return "", nil
+	}
+	if err := json.Unmarshal(*manifest.VendorLink, &link); err != nil {
+		return "", err
+	}
+	link = strings.Trim(link, "'")
+	return link, nil
+}
+
+func updateSettings(inst *instance.Instance, attempt int, logger *logrus.Entry) error {
+	logger.Infof("Updating bitwarden settings after migration...")
+	// Reload the setting in case the revision changed
+	setting, err := settings.Get(inst)
+	if err != nil {
+		return err
+	}
+	// This flag is checked at the extension pre-login to run the migration or not
+	setting.ExtensionInstalled = true
+	err = settings.UpdateRevisionDate(inst, setting)
+	if err != nil {
+		if couchdb.IsConflictError(err) && attempt < 2 {
+			err = updateSettings(inst, attempt+1, logger)
+			if err != nil {
+				return err
+			}
```

Comment:
You can simplify those 4 lines in just `return updateSettings(inst, attempt+1, logger)`